public/javascripts/redirect.js

```diff
@@ -0,0 +1,74 @@
+// Redirect all the existing URLs with fragment identifiers in to the new section
+(function () {
+  'use strict';
+
+
+  var getNewRouteFor = function (fragment) {
+    var baseURL = 'http://govuk-elements.herokuapp.com';
```

Comment:
Can this be changed to the site origin so the redirects will work outside of heroku?

I was thinking something like: `var baseURL = window.location.href.split('#')[0];`